contract-map.json

```diff
@@ -1,4 +1,11 @@
 {
+  "0x00c83aeCC790e8a4453e5dD3B0B4b3680501a7A7": {
+    "name": "SKALE",
+    "logo": "skl.svg",
+    "erc20": true,
```

Comment:
The SKL token is an ERC777.

Should the `erc20` value be false?

Reply:
erc-777 is backward compatible with erc-20